front/computer_softwareversion.form.php

```diff
@@ -49,6 +49,15 @@
                     //TRANS: %s is the user login
                     sprintf(__('%s installs software'), $_SESSION["glpiname"]));
       }
+   } else {
+       $message = null;
+       if (!isset($_POST['softwares_id']) || !$_POST['softwares_id']) {
```

Comment:
Maybe I missed something, but I think `softwares_id` is not used here. Did you mean `computers_id` ?

Reply:
You have to select a software (`softwares_id`) and then a version (`softwareversions_id`). This is indeed not used directly, but it's not possible to select a version if you did not select a software.

Reply:
Page is called from a computer; si this is not possible to have an empty id. User have to select a soft and its version, and can click if it's empty.

Reply:
ok